### DISCUSSION TITLE
Substation automation using IEC 61850 standard

### DISCUSSION THREAD

**Discussion by rakeshg83** [riclolsen/json-scada#40] [General]
We are developing an application for Substation automation using IEC 61850 standard. Whether we can use this json SCADA or OSHMI?
Please suggest

**Comment by riclolsen** [comment]
Hi.
This protocol is not currently available for JSON-SCADA nor for OSHMI.
IEC61850 client is planned for JSON-SCADA, there are no target dates but it can possibly be implemented in 2022.
For OSHMI, there is no plan to support IEC61850.
Best regards,
Ricardo

**Reply by rakeshg83** [reply]
Thanks sir for your reply.
We are developing a substation automation system based on IEC 61850 and IEC 62351 (security protocols) standards. I am from INDIA. We are interested to colloborate on your work.  Can you share your contact details. I am Rakesh G, ScientistE, C-DAC, Trivandrum, INDIA. Ours is a is a government undertaking organization . Kindly contact me on rakeshg@cdac.in or in g83.rakesh@gmail.com

Thanks in advance

**Reply by rakeshg83** [reply]
Ours is a national level project called ""Digital substation automation system for smart power quality centre "

**Comment by riclolsen** [comment]
IEC 61850 is now available for OSHMI and JSON-SCADA.
